frontend/src/app/main/data/workspace/text/shortcuts.cljs

```diff
@@ -0,0 +1,218 @@
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:text-align "right"})}
+   :align-center  {:tooltip (ds/meta (ds/alt "t"))
+                   :command (ds/c-mod "alt+t")
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:text-align "center"})}
+   :align-justify {:tooltip (ds/meta (ds/alt "j"))
+                   :command (ds/c-mod "alt+j")
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:text-align "justify"})}
+
+   :underline     {:tooltip (ds/meta "u")
+                   :command (ds/c-mod "u")
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:text-decoration "underline"})}
+
+   :line-through  {:tooltip (ds/alt (ds/meta-shift "5"))
+                   :command "alt+shift+5"
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:text-decoration "line-through"})}
+
+   :font-size-inc {:tooltip (ds/meta-shift ds/up-arrow)
+                   :command (ds/c-mod "shift+up")
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:font-size-inc true})}
+
+   :font-size-dec {:tooltip (ds/meta-shift ds/down-arrow)
+                   :command (ds/c-mod "shift+down")
+                   :subsections [:text-editor]
+                   :fn #(update-attrs-when-no-readonly {:font-size-dec true})}
+
+   :line-height-inc {:tooltip (ds/alt-shift ds/up-arrow)
+                     :command (ds/a-mod "shift+up")
+                     :subsections [:text-editor]
+                     :fn #(update-attrs-when-no-readonly {:line-height-inc true})}
+
+   :line-height-dec {:tooltip (ds/alt-shift ds/down-arrow)
+                     :command (ds/a-mod "shift+down")
+                     :subsections [:text-editor]
+                     :fn #(update-attrs-when-no-readonly {:line-height-dec true})}
+
+   :letter-spacing-inc {:tooltip (ds/alt ds/up-arrow)
+                        :command (ds/a-mod "up")
+                        :subsections [:text-editor]
+                        :fn #(update-attrs-when-no-readonly {:letter-spacing-inc true})}
+
+   :letter-spacing-dec {:tooltip (ds/alt ds/down-arrow)
+                        :command (ds/a-mod "down")
+                        :subsections [:text-editor]
+                        :fn #(update-attrs-when-no-readonly {:letter-spacing-dec true})}
+
+   :bold     {:tooltip (ds/meta "b")
+              :command (ds/c-mod "b")
+              :subsections [:text-editor]
+              :fn #(update-attrs-when-no-readonly {:font-variant-id "bold"})}
+
+   :italic     {:tooltip (ds/meta "i")
+                :command (ds/c-mod "i")
+                :subsections [:text-editor]
+                :fn #(update-attrs-when-no-readonly {:font-variant-id "italic"})}}
+
+
+)
```

Comment:
Move this end parenthesis to the previous element

Reply:
fixed